README.md

```diff
@@ -52,14 +52,18 @@ Install the MAU bundle (or, starting a dev session every time)
     git checkout master
     git pull
     bundle install
+    
+Note that on MacOS Mojave, you may need to run the following command in order for the `mysql` gem to be successfuly installed and the `bundle install` to complete. See https://gist.github.com/fernandoaleman/ee3ac6957c2ba4f7d7d33a251d58b191 for original instructions and for link to potential fix for the same issue on Catalina.
+
+    gem install mysql2 -v '0.5.3' -- --with-ldflags=-L/usr/local/opt/openssl/lib --with-cppflags=-I/usr/local/opt/openssl/include
 
 Tell Rails to build you an empty database
 
     rake db:create:all # only if you cloned the repo
 
-Tell Rails to setup that database and build a copy of that schema for testing
+Tell Rails to load the schema for the database and build a copy of that schema for testing
 
-    rake db:migrate db:test:prepare
+    rake db:schema:load db:test:prepare
```

Comment:
Interestingly, I think the *actual* right approach here is to run `bin/setup` which does `bundle install` and `db:schema:load`.  It doesn't change the notes above about the bundle issue which you'd likely hit the same way but this might be a better approach.  We can either merge this as is and I'll make the changes later or not.

I took a few minutes to look into squashing all the migrations into a new single migration but that lead to it's own issues so that may not happen.

Reply:
So something like this?
```suggestion
    ./bin/setup
    rake db:test:prepare
```

(See the comment above as well updating the instructions to reflect this)

Reply:
I think that's right.  

if you look [here](https://edgeguides.rubyonrails.org/active_record_migrations.html#setup-the-database) it looks like `db:setup` does (under the hood) `db:create` `db:schema:load`. so that's the right starting point.

Reply:
I kinda forgot I could just check the rails guide 😅 so if this looks good I'll make the change here and above and then merge.

Reply:
Sounds good!